apps/trading/components/navbar/nav-header.tsx

```diff
@@ -0,0 +1,67 @@
+import { VegaIcon, VegaIconNames } from '@vegaprotocol/ui-toolkit';
+import { MarketSelector } from '../market-selector';
+import { useMarket } from '@vegaprotocol/markets';
+import { t } from '@vegaprotocol/i18n';
+import { useParams } from 'react-router-dom';
+import * as PopoverPrimitive from '@radix-ui/react-popover';
+import { useState } from 'react';
+
+/**
+ * This is only rendered for the mobile navigation
+ */
+export const NavHeader = () => {
+  const { marketId } = useParams();
+  const { data } = useMarket(marketId);
+  const [open, setOpen] = useState(false);
+
+  if (!marketId) return null;
+
+  return (
+    <FullScreenPopover
+      open={open}
+      onOpenChange={(x) => {
+        console.log(x);
```

Comment:
We should add no-console eslint rule long time ago

Reply:
Added a default console log rule (which still allows warn/error)